Dockerfile.webhook

```diff
@@ -0,0 +1,27 @@
+ARG TAG="release-4.8"
+ARG UBI_IMAGE=registry.access.redhat.com/ubi7/ubi-minimal:latest
+ARG GO_IMAGE=rancher/hardened-build-base:v1.15.8b5
+
+FROM ${GO_IMAGE} as builder
+RUN set -x \
+ && apk --no-cache add \
+    git \
+    make \
+    patch
+ARG TAG
+RUN git clone https://github.com/k8snetworkplumbingwg/sriov-network-operator
+WORKDIR sriov-network-operator
+RUN git fetch --all --tags --prune
+RUN git checkout remotes/origin/${TAG} -b ${TAG}
+COPY 0001-CGO_ENABLED.patch 0002-Change-the-module-URL-from-openshift-to-k8snetworkpl.patch ./
+RUN patch -p1 < 0001-CGO_ENABLED.patch
+RUN patch -p1 < 0002-Change-the-module-URL-from-openshift-to-k8snetworkpl.patch
+ENV CGO_ENABLED=0
+RUN make _build-webhook
```

Comment:
Does the `_build-webhook` target also install the binary in `/usr/bin`? It seems like, but that's really cofusing...

Reply:
It installs the binary in build/_output/linux/amd64/webhook. Then we copy that to `/usr/bin/webhook` in the UBI_IMAGE